docs/destinations/warehouses-and-lakes/index.md

```diff
@@ -3,8 +3,40 @@ title: "Data warehouses and lakes"
 sidebar_position: 2
 ---
 
-```mdx-code-block
-import DocCardList from '@theme/DocCardList';
+To store your enriched Snowplow data, you will need to determine which loader to set-up to ensure your enriched data reaches your data warehouse or lake. 
 
-<DocCardList />
-```
+### Selecting a loader for your data warehouse
+#### Selecting a loader by destination and cloud provider
+
+The Snowplow pipeline supports loading into several data warehouses in AWS and GCP, and the support varies by cloud provider.
+
+##### Data warehouse destinations supported in the Snowplow pipeline for AWS
+| Destination | Loader | Status |
+| --- | --- | --- |
+| Redshift | [Snowplow RDB Loader](/docs/destinations/warehouses-and-lakes/rdb/index.md) | Production-ready |
+| Snowflake | [Snowplow RDB Loader](/docs/destinations/warehouses-and-lakes/rdb/index.md) | Production-ready |
+| Databricks | [Snowplow RDB Loader](/docs/destinations/warehouses-and-lakes/rdb/index.md) | Production-ready |
+| Postgres | [Postgres Loader](/docs/destinations/warehouses-and-lakes/postgres/index.md) | Early release |
+
+##### Data warehouse destinations supported in the Snowplow pipeline for GCP
+
+| Destination | Loader | Status |
+| --- | --- | --- |
+| Snowflake | [Snowplow RDB Loader](/docs/destinations/warehouses-and-lakes/rdb/index.md) | Production-ready |
+| BigQuery | [BigQuery Loader](/docs/destinations/warehouses-and-lakes/bigquery/index.md) | Production-ready |
+| Postgres | [Postgres Loader](/docs/destinations/warehouses-and-lakes/postgres/index.md) | Early release |
+
+#### Selecting a loader by stream processing technology support
```

Comment:
If you have a better term than "stream processing technology" please let me know 🙏

Reply:
It seemed valuable to me to explain that if you take advantage of the broader support available for Kafka/RabbitMQ/NSQ in the Collector or Enrich, that currently you won't be able to simply configure the loaders to read from those. But admittedly, I don't know how broadly that Kafka/RabbitMQ/NSQ support is utilised. What about replacing the whole "Selecting a loader by stream processing technology support" section with:

### If you are not using Kinesis or Pub/Sub

Other stream processing technologies are supported by the [Stream Collector](/docs/pipeline-components-and-applications/stream-collector/index.md) and [Enrich](/docs/pipeline-components-and-applications/enrichment-components/index.md), but if you prefer to use a stream processing technology other than Kinesis or Pub/Sub, currently you will need to define your own process to load your enriched data into your data warehouse.


?